lab-2.ipynb

```diff
@@ -0,0 +1,1233 @@
+      "text/plain": [
+       "<Figure size 720x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# day 2 (one day being added, semilog)\n",
+    "\n",
+    "# convolve background (Poisson distribution) once with itself\n",
+    "plt.rcParams['figure.figsize'] = [10, 5]\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "plt.yscale('log')\n",
+    "ax.step(num_events2, background_day_2, where = 'mid', label = \"Cosmic Background\") \n",
+    "plt.title(\"Poisson Distribution of Cosmic Background (semi-logY) (DAY 2)\", size = 15)\n",
+    "plt.xlabel(\"Number of Events\", size= 15)\n",
+    "plt.ylabel(\"log(Probability)\", size=15)\n",
+    "\n",
+    "mu = 10 # because the day has advanced, average has increased (sum of two)\n",
+    "def f(x):\n",
+    "   return np.exp(-mu)*(mu**x)/(scipy.special.factorial(x)) # Poisson distribution (the actual function)\n",
+    "x = np.linspace(0, 35, 1000)\n",
+    "\n",
+    "ax.plot(x, f(x), color='red', label=\"Continuous pmf()\")\n",
+    "plt.xticks(fontsize = 12)\n",
+    "plt.yticks(fontsize = 12)\n",
+    "plt.legend(loc=\"best\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8997fc94-febf-4a8e-bed3-4bc4f85bbe49",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### Day 3 (one day is added)"
```

Comment:
I would lengthen the range to a larger # of days. perhaps 10 and 100 instead of 3 or 4? that way it can show what happens at a greater number of days